src/app/model.ts

```diff
@@ -128,25 +128,191 @@ export interface Step extends BaseEntity {
 export type Steps = Array < Step > ;
 
 export interface Tag extends BaseEntity {
-    integrationTemplate: Array < IntegrationTemplate >
-    ;
     connections: Array < Connection >
     ;
+    integrationTemplate: Array < IntegrationTemplate >
+    ;
     id: string;
     name: string;
 };
 export type Tags = Array < Tag > ;
 
 export interface User extends BaseEntity {
-    fullName: string;
     username: string;
+    fullName: string;
+    organizationId: string;
     lastName: string;
     firstName: string;
     integrations: Array < Integration >
     ;
     roleId: string;
-    organizationId: string;
     name: string;
     id: string;
 };
-export type Users = Array < User > ;
\ No newline at end of file
+export type Users = Array < User > ;
+
+export interface EventMessage extends BaseEntity {
+    data: {};
+    event: string;
+};
+export type EventMessages = Array < EventMessage > ;
+
+class TypeFactoryClass {
+    createListResult() {
+        return <ListResult > {
+            items: undefined,
+            totalCount: undefined,
+        };
+    };
+
+    createListResultWithId() {
+        return <ListResultWithId > {
+            items: undefined,
+            totalCount: undefined,
+        };
+    };
+
+    createWithId() {
+        return <WithId > {
+            id: undefined,
+        };
+    };
+
+    createConnection() {
+        return <Connection > {
+            icon: undefined,
+            organization: undefined,
+            tags: undefined,
+            position: undefined,
+            description: undefined,
+            connector: undefined,
+            userId: undefined,
+            configuredProperties: undefined,
+            connectorId: undefined,
+            organizationId: undefined,
+            id: undefined,
+            name: undefined,
+        };
+    };
+
+    createConnector() {
+        return <Connector > {
+            icon: undefined,
+            description: undefined,
+            connectorGroup: undefined,
+            connectorGroupId: undefined,
+            properties: undefined,
+            id: undefined,
+            name: undefined,
+        };
+    };
+
+    createConnectorGroup() {
+        return <ConnectorGroup > {
+            id: undefined,
+            name: undefined,
+        };
+    };
+
+    createEnvironment() {
```

Comment:
Wondering if it would make sense to break these up into files in the future as we add more models.